Zend/tests/fibers/fiber-this.phpt

```diff
@@ -1,12 +1,12 @@
 --TEST--
-Fiber::this()
+Fiber::getCurrent()
```

Comment:
Would it be worth to also rename this file?

Reply:
Oops, missed that, thank you for pointing this out. Updated now.